templates/patterns/file_translation/pattern-file-translation.html.twig

```diff
@@ -62,5 +67,7 @@
     },
     'description': more_info|default(''),
     'items': _translations,
-  }
+  },
+  'variant': 'thumbnail',
```

Comment:
The problem here is that this is not "backwards compatible", in the sense that if someone is using this pattern and does not add a thumbnail will get nothing. I propose we check whether we have a thumbnail or not and set the variant accordingly? cc @ademarco